src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
```

Comment:
Only issue with axing `scope.ttOffset` is that it's used very specifically related to the positioning desired (left vs right), etc. Sorry for the back and forth on this, we may need to retain it as it's effectively acting as a default padding / margin. 

So here's my final suggestion: 

We retain the existing offset functionality, but rename it to "margin" instead, so as not to be confusing with the new offset properties. So the only changes required would be renaming things and clarifying the documentation; the calculations would remain as they are in your PR.

I don't think that we'd need to expose the vertical / horizontal offsets as global config values, because I can't see an immediate use case where all tips would need positioning adjustments ... but maybe i'm missing a use-case.

Sound good?